static/templates/settings/organization-permissions-admin.handlebars

```diff
@@ -95,11 +95,14 @@
 
 
             <div class="input-group disableable">
-                <label for="aitin">{{t "Waiting period for stream creation (in days)" }}</label>
-                <input type="text" id="id_realm_waiting_period_threshold"
-                     name="realm_waiting_period_threshold"
-                     class="admin-realm-message-content-edit-limit-minutes"
-                     value="{{ realm_waiting_period_threshold }}"/>
+                <label for="aitin">
+                    {{t "Prevent new users from creating stream for" }}
+                    <input type="text" id="id_realm_waiting_period_threshold"
+                         name="realm_waiting_period_threshold"
+                         class="admin-realm-message-content-edit-limit-minutes"
+                         value="{{ realm_waiting_period_threshold }}"/>
+                    {{t "(days)" }}
```

Comment:
This structure unfortunately can't work for translation, since in other languages the "3 days" might appear in a different part of the sentence.

Reply:
Hmm, right let me find better solution.